documentation/src/main/asciidoc/query.asciidoc

```diff
@@ -289,25 +289,54 @@ Query luceneQuery = mythQB.keyword()
 In the previous example, only field name is boosted to 5.
 
 
-===== Simple query string queries
+===== Simple Query String queries
 
-Simple query string queries use the Lucene SimpleQueryParser. The query text is parsed and it exposes more
-search features to the end user such as boolean operators and phrase queries.
+Simple Query String queries use the Lucene `SimpleQueryParser`, presented as a "parser for
+human-entered queries". The point of this parser is to be a very simple lenient state machine
+to parse queries entered by your end users.
+
+The parser is capable to transform `keyword "some phrase" -keywordidontwant fuzzy~ prefix*`
+into a Lucene query, giving your users a little more power (phrase queries, fuzzy queries,
+boolean operators...).
+
+The lenient part is important as it will try to build the best possible query without throwing
+a parsing exception, even if the query is not what we would consider syntactically correct.
+
+Another nice feature is that it allows to search on multiple fields. You basically end up
+establishing the following contract with Lucene:
+
+ * users will enter a search query (more or less syntactically correct)
+ * it will search on the fields you have specified (and you can also specify a specific
+   boost for each field)
+ * you can enable each of the features that you want to expose to the users
+   (i.e. you can enable the phrase queries but not the boolean operators)
+ * building the query won’t throw an exception
+
+[NOTE]
+====
+Even if we are talking about Lucene queries here, this feature is also supported by our
+Elasticsearch integration.
+====
+
+Here are a couple of examples introducing the features of our Simple Query String support one at a time.
+
+The query below is the most simple form of Simple Query String query. It simply returns documents
+whose `history` field contains `storm`:
 
 ====
 [source, JAVA]
 ----
 Query luceneQuery = mythQB
     .simpleQueryString()
     .onField("history")
-    .matching("(storm OR hurricane) AND \"falling branches\"")
+    .matching("storm")
     .createQuery();
 ----
 ====
 
-By default, a simple query string query uses the OR operator if the operator is not explicitly defined.
+By default, a Simple Query String query uses the OR operator if the operator is not explicitly defined.
```

Comment:
+1 let's keep it consistently capitalized to clarify that it's the proper name of the feature.